js/templates/modals/editListing/variantInventory.html

```diff
@@ -3,15 +3,15 @@
 <% } else { %>
 <div class="inventoryTableWrap">
   <table>
-    <tr>
+    <thead>
       <% ob.columns.forEach(column => { %>
-      <th class="clrBr"><%= column %></th>
+      <td class="clrBr"><div class="stickyDiv"><%= column %></div></td>
       <% }) %>
-      <th class="clrBr">Surcharge</th>
-      <th class="clrBr">Total Price</th>
-      <th class="clrBr">Sku</th>
-      <th class="clrBr quantityCol">Quantity</th>
-    </tr>
+      <td class="clrBr"><div class="stickyDiv">Surcharge</div></td>
```

Comment:
From the examples I've seen, I think these should be ths, even when they are in a thead. That would give you the bolding.

Reply:
Chrome does not like the `position: sticky;` attribute on TH. THEAD can contain any valid TR. TR can contain either TH or TD. TH is used to mark the cells as header cells specifically, but using TD is not invalid. Source: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead 

Will restore bolding by applying correct classes/adding a bold attribute.